### PR TITLE
Update installation.markdown (virtual NIC notes for ESXi/vSphere installs to avoid mDNS/Multicast discovery issues)

### PR DESCRIPTION
**Pull request by MarkHofmann11**
The update is regarding this recent discovery with mDNS and cast:  https://github.com/home-assistant/core/issues/43652

This is the section that has the relevant information:

     - **For VMware ESXi/vSphere/VMware Workstation installs - use the "E1001" or "E1001E" virtual network adapater.**  There are confirmed mDNS/Multicast discovery issues when using VMware's "VMXnet3" virtual network adapter.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -29,6 +29,7 @@ The following will take you through the steps required to install Home Assistant
...
      - For KVM create a new virtual machine in `virt-manager`, select "Import existing disk image", provide the path to the QCOW2 image above, choose "Generic Default" for the operating system, assign at least 2 GB memory and 1 vCPU, check the box for "Customize configuration before install" and select your bridge under "Network Selection", then under customization select "Overview" -> "Firmware" -> "UEFI x86_64: ...".
      - For Vmware Workstation create a new virtual machine, select "Custom", make it compatible with the default of Workstation and ESX, Choose "I will install the operating system later", select "Linux" -> "Other Linux 5.x or later kernel 64-bit", give it at least 2 GB RAM and 1vCPU, select "Use Bridged Networking" then "Use an existing virtual disk" and select the VMDK file above, after creation of VM go to "Settings" and "Options" then "Advanced" and select "Firmware type" to "UEFI".
+     - **For VMware ESXi/vSphere/VMware Workstation installs - use the "E1001" or "E1001E" virtual network adapater.**  There are confirmed mDNS/Multicast discovery issues when using VMware's "VMXnet3" virtual network adapter.
```
Comment (fabaff) [inline source/hassio/installation.markdown]: This should be a list entry. Add it to the "Vmware Workstation" section please.

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/hassio/installation.markdown (hunk 1 above)
Comment (MarkHofmann11) [reply source/hassio/installation.markdown]: Just discovered that it doesn't appear that VMware Workstation has the E1001 or E1001E vNICs, so this really only applies to ESXi/vSphere installs.  Would it be ok to leave it where it is, like this:

For VMware ESXi/vSphere installs - use the "E1001" or "E1001E" virtual network adapater.**  There are confirmed mDNS/Multicast discovery issues when using VMware's "VMXnet3" virtual network adapter.

**Review by MarkHofmann11 — COMMENTED** [review]
This only applies to VMware ESXI/vSphere installs, so removed the part about VMware workstation.

File: source/hassio/installation.markdown (hunk 1 above)
Comment (fabaff) [reply source/hassio/installation.markdown]: Then it should stay where it is.

**Review by fabaff — APPROVED** [review]

**Pull request merged**